src/lib/api.js

```diff
@@ -631,7 +645,40 @@ const searchItems = async function (collectionId, queryParameters, backend, endp
         results: []
       }
     } else {
-      throw error
+      try {
+        // Parse some types of geometry errors from Search
+        // @ts-ignore
+        const reason1 = error['meta']['body']['error']['caused_by'][0]['reason']
+        return {
+          statusCode: 400,
+          body: reason1
+        }
+      } catch (e1) {
```

Comment:
nit, feel free to ignore: since the error is unused, can we use `_` as the variable name to show that its not used? Unsure if this works in javascript-land, haven't been here in a while :-) 🙇🏼 

```suggestion
      } catch (_) {
```